artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/RemotingConnectionImpl.java

```diff
@@ -421,6 +434,34 @@ private void doBufferReceived(final Packet packet) {
       }
    }
 
+   private boolean isCorrectPing(Packet packet) {
+      if (packet.getType() != PING) {
+         return true;
```

Comment:
Commenting this line out will effectively disable the fix. This will cause the new test `ZeroPersistenceSymmetricalClusterTest` to fail.